src/soc/intel/meteorlake/romstage/fsp_params.c

```diff
@@ -165,7 +167,7 @@ static void fill_fspm_mrc_params(FSP_M_CONFIG *m_cfg,
 	m_cfg->RMC = 0;
 	m_cfg->MarginLimitCheck = 0;
 	/* Enable MRC Fast Boot */
-	m_cfg->MrcFastBoot = 1;
+	m_cfg->MrcFastBoot = !vbnv_cmos_failed();
```

Comment:
AFAICT this won't compile without `CONFIG_VBOOT`, and there is nothing in Kconfig for SoC that enforces it to be enabled. In any case, I don't think this is a good plate to add dependency on vboot to.

Reply:
It doesn't actually depend on Vboot. The function is defined in https://github.com/Dasharo/coreboot/blob/691bec1b71bdb814fef1f55fe175a19af6923aab/src/soc/intel/meteorlake/pmutil.c#L179-L182 and is compiled regardless if Vboot is enabled or not.

Reply:
Right, I only checked that it is declared in  https://github.com/Dasharo/coreboot/blob/dasharo/src/security/vboot/vbnv.h#L27, my bad.